### DISCUSSION TITLE
Is there a way to output prisma to two directories?

### DISCUSSION THREAD

**Discussion by izaacdb** [prisma/prisma#8932] [Q&A] [answered]
I'm generating Prisma like this:

````
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["filterJson"]
  output   = "../shared/prisma"
}
````

This has some issues with other frameworks (https://github.com/nextauthjs/adapters/issues/191) so I'd like to generate two copies at once, something like this:

````
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["filterJson"]
  output   = ["../server/node_modules/.prisma",'../frontend/node_modules/.prisma']
}
````

Is this possible without writing a script to do it?

**Comment by janpio** [comment, marked as answer]
Right now `output` can only be a string. This sounds like an interesting feature request though, so please open one here in the repository and we might prioritize this in the future.

**Reply by ryands17** [reply]
@izaacdb 👋 

You can open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
